cpp/include/IceSSL/IceSSL.h

```diff
@@ -5,9 +5,6 @@
 #ifndef ICESSL_ICESSL_H
 #define ICESSL_ICESSL_H
 
-#include "Config.h"
-#include "Plugin.h"
-
 #if defined(_WIN32)
```

Comment:
I assume we want to remove the IceSSL include directory entirely once the merger is complete?

Reply:
Yes, the remaining public headers:

```
Certificate.h 
ConnectionInfo.h 
ConnectionInfoF.h 
EndpointInfo.h
OpenSSL.h  
SChannel.h 
SecureTransport.h
```